calm-visualizer/src/components/graph/Graph.tsx

```diff
@@ -29,7 +27,8 @@ function createConnectsRelationship(instance: BrowserJsPlumbInstance, relationsh
             options: {}
         },
         overlays: [
-            { type:"Arrow", options:{location:1}}
+            { type:"Arrow", options:{location:1}},
+            {  type: "Label", options: { location: 0.5, label: relationship.description }}
```

Comment:
Will this cause any problems if there isn't a description defined on the relationship?

Reply:
label also accepts undefined as a value in the case the description isnt defined